src/book/02-system/03-tools/05-system-configuration.mdx

```diff
@@ -0,0 +1,60 @@
+---
+section: System
+chapter: Tools
+title: System Configuration
+description: A tool that sets up the robot.
+slug: /system/tools/system_configuration
+---
+
+## Description
+
+This module is used to ensure that the configuration that is running on a device matches what is required.
+
+It goes through several steps to ensure the system is set up correctly.
+
+### Filesystem
+
+The system will look through each file in the `module/tools/SystemConfiguration/data/system/default` directory.
+For each of these files it will see if there is an equivalent file in the
+`module/tools/SystemConfiguration/data/system/$HOSTNAME` directory and use that, otherwise it uses the default file.
+Using this file it will check that the same file in the root directory is identical to this one, and if not it will copy it across.
+This will ensure that all the system configuration files match the files that are in the repository.
+
+<Alert type='info'>
+The system will only look at files that exist in the `module/tools/SystemConfiguration/data/system/default` directory.
```

Comment:
If you want Markdown handling in the Alert (to e.g. highlight the backtick code bits) you'll need a new line after the opening tag (and another before the closing one below as well).

```suggestion
<Alert type='info'>

The system will only look at files that exist in the `module/tools/SystemConfiguration/data/system/default` directory.
```